pkg/controller/awsprivatelink/awsprivatelink_controller.go

```diff
@@ -267,6 +259,15 @@ func (r *ReconcileAWSPrivateLink) Reconcile(ctx context.Context, request reconci
 				}
 				return reconcile.Result{}, err
 			}
+
+			if err := r.setProgressCondition(cd, corev1.ConditionFalse,
+				"PreviousAttemptCleanupComplete",
```

Comment:
`PreviousAttemptedCleanupComplete` may be?

Reply:
Seems a little off, that makes it sound like a previous attempted cleanup, rather than a cleanup for a previous attempted provision. 

What about ProvisionAttemptCleanupComplete?

Reply:
My bad. Lets stick with `PreviousAttemptCleanupComplete`